urlsearchparams/index.html

```diff
@@ -0,0 +1,116 @@
+---
+feature_name: URLSearchParams
+chrome_version: 49
+feature_id: 5632984866619392
+local_css_files: ['page.css']
+additional_head_content: <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Product+Sans">
```

Comment:
+1 to @beaufortfrancois comment, in terms of leaving this out. The samples pages are rather bare-boned, stylistically, but they're all consistent.

If you'd want to prettify the layout or typography, it would best be done against the master templates and stylesheet so that it would apply to all the samples in a consistent manner (or at least the subset of samples that have moved to the templates).